src/js/edu-benefits/components/personal-information/DependentInformationFields.jsx

```diff
@@ -21,7 +16,7 @@ export default class DependentInformationFields extends React.Component {
             value={this.props.data.serviceBefore1977.married}
             onValueChange={(update) => {this.props.onStateChange('serviceBefore1977.married', update);}}/>
         <ErrorableRadioButtons
-            label="Do you have any children who are under age 18, or over age 18 but under age 23, not married and attending school, or of any age permanently helpless for mental or physical reasons?"
+            label="Do you have any children who are under age 18, or over age 18 but under 23, not married and attending school, or of any age permanently disabled for mental or physical reasons?"
```

Comment:
I wonder if we should list out these conditions in bullets instead of listing them all in a single sentence. It's hard to understand how certain things are grouped. For example, I had to read this several times to understand that `over age 18 but under 23, not married and attending school` is a group.